Camunda.Api.Client/UserTask/CompleteTaskBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Camunda.Api.Client.UserTask
+{
+    public abstract class CompleteTaskBase
+    {
+        /// <summary>
+        /// Variables will return after completing the task
+        /// </summary>
+        public abstract bool WithVariablesInReturn { get; }
+
+        /// <summary>
+        /// Object containing variable key-value pairs.
+        /// </summary>
+        public Dictionary<string, VariableValue> Variables = new Dictionary<string, VariableValue>();
```

Comment:
Is it better to use a property with a private setter?

Reply:
I think this is incompatible